Creating Header and Headlines sections of Portfolio

NitBravoA92 opened the pull request.

NitBravoA92 committed:
Create linters.yml

NitBravoA92 committed:
Add node_modules folder

NitBravoA92 committed:
Initialize npm to create package.json file

NitBravoA92 committed:
Install linters libraries

NitBravoA92 committed:
Create .hintrc file

NitBravoA92 committed:
Create .stylelintrc file

NitBravoA92 committed:
Create index.html file for Homepage

NitBravoA92 committed:
Create style.css

NitBravoA92 committed:
Add images and icons to img folder

NitBravoA92 committed:
Include The style.css File Inside The Head Section

NitBravoA92 committed:
Add fonts external link

NitBravoA92 committed:
Fix indentation in the code

NitBravoA92 committed:
Create skeleton header section

NitBravoA92 committed:
Create the skeleton of the Headline section

NitBravoA92 committed:
Fix indentation issues in the code

NitBravoA92 committed:
Create general css styles

NitBravoA92 committed:
Fix hexadecimal color sintax error

NitBravoA92 committed:
Create css styles for header section

NitBravoA92 committed:
Create css styles for headline section

NitBravoA92 committed:
Add css styles for social media links

NitBravoA92 committed:
Create first media query for Desktop version

NitBravoA92 committed:
Fix indentation errors

NitBravoA92 committed:
Update README.md file

hiromataba commented on index.html:
```diff
@@ -0,0 +1,55 @@
+  <body>
+    <header>
+      <div class="logo-container">
+        <h1 class="logo">My Logo</h1>
```
- [ ]  Please update your logo to be a link (`a` tag) because when clicked on it it should take us to the main page

hiromataba commented on index.html:
```diff
@@ -0,0 +1,55 @@
+        <div class="section-mobile-bg"></div>
+        <div class="section-mobile-shape"></div>
+          <div class="container">
+            <h2 class="main-greeting">
```
- [ ] Please update your main title to `h1` because this is actually a main title

hiromataba commented on index.html:
```diff
@@ -0,0 +1,55 @@
+            </p>
+            <div class="social-media">
+              <p class="connect-message">LET’S CONNECT</p>
+              <ul class="social-media-links">
+                <li><a href="#"><img src="./img/github-icon.png" alt="github icon"></a></li>
+                <li><a href="#"><img src="./img/linkedin-icon.png" alt="linkedin icon"></a></li>
+                <li><a href="#"><img src="./img/angelist-icon.png" alt="angelist icon"></a></li>
+                <li><a href="#"><img src="./img/twitter-icon.png" alt="twitter icon"></a></li>
+                <li><a href="#"><img src="./img/medium-icon.png" alt="medium icon"></a></li>
+              </ul>
```
- [x] Nice job using `ul` tag to list your social icons 👍  🚀

hiromataba reviewed (changes requested):
Hi @NitBravoA92    :wave: :wave: 

![almost-there](https://user-images.githubusercontent.com/75126481/160092588-b3f37e8e-a1d9-4b14-84fc-7ec1fed7b3e5.gif)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NitBravoA92 committed:
Update main title and logo section

Change the main title to an h1 tag and change the logo to be a link to the homepage

ridwanediallo commented on index.html:
```diff
@@ -11,7 +11,7 @@
     <header>
       <div class="logo-container">
-        <h1 class="logo">My Logo</h1>
+        <h1 href="index.html" class="logo">My Logo</h1>
```
- [ ] I think you forgot to change the `h1` tag to an `a`  🤓. Please ensure to fix that because I believe this is not a valid way to add a link. :ok_hand: 
```suggestion
<a href="index.html" class="logo">My Logo</a>
```

ridwanediallo reviewed (changes requested):
# Changes Required ♻️
## Hi @NitBravoA92 , 


![AlmostThereReachGIF](https://user-images.githubusercontent.com/55918904/191301799-36d07166-0e35-4582-a231-7a2d780f7783.gif)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You used GitHub flow correctly. ✔
- Your commit messages are descriptive. ✔


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NitBravoA92 committed:
Update logo section

Change the logo to be an anchor element

V-Blaze commented on index.html:
```diff
@@ -0,0 +1,55 @@
+            </p>
+            <div class="social-media">
+              <p class="connect-message">LET’S CONNECT</p>
+              <ul class="social-media-links">
+                <li><a href="#"><img src="./img/github-icon.png" alt="github icon"></a></li>
+                <li><a href="#"><img src="./img/linkedin-icon.png" alt="linkedin icon"></a></li>
+                <li><a href="#"><img src="./img/angelist-icon.png" alt="angelist icon"></a></li>
+                <li><a href="#"><img src="./img/twitter-icon.png" alt="twitter icon"></a></li>
+                <li><a href="#"><img src="./img/medium-icon.png" alt="medium icon"></a></li>
+              </ul>
```
- [OPTIONAL] Just as the previous reviewer said, you did a great job using the `ul` and the `a` tag here. I'll suggest you add the actual `href` links to your social media profiles here. It will make it easier for people to browse through your projects and reach about to you. :taco: :+1: :zap:

NitBravoA92 replied:
Hi, @V-Blaze Thank you so much for those suggestions, I'll take them into account to improve my project and my professional skills.

V-Blaze reviewed (approved):
Hi @NitBravoA92  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![giphy](https://user-images.githubusercontent.com/107632780/229644741-7dea3027-bf3f-4416-855b-46095e3bf6d5.gif)

## Highlights

- Project looks like the given design💯
-  Code clean and used correct semantic tags👍
-  Readme is detailed and looks professional🟢 👍

## Optional suggestions

- [OPTIOANL] This wasn't pointed out by the previous reviewers, I noticed your pull request doesn't have a summary. I'll suggest you edit the pull request and add a descriptive summary, Summary should Include the major features you added or developed in this current branch. :+1: :100: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NitBravoA92 committed:
Update index.html

Add social media account URLs to the links in the Headline section
